host/testcluster.go

```diff
@@ -335,13 +335,14 @@ func newArchiverBase(enabled bool, logger log.Logger) *ArchiverBase {
 		FileMode: "0666",
 		DirMode:  "0766",
 	}
-	provider := provider.NewArchiverProvider(
-		&config.HistoryArchiverProvider{
-			Filestore: cfg,
-		},
-		&config.VisibilityArchiverProvider{
-			Filestore: cfg,
-		},
+	node, err := config.ToYamlNode(cfg)
```

Comment:
nice

Reply:
not my proudest bit of shenaniganry, I'll admit, but it is at least pretty simple :)